taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from .models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver

```

Comment:
<img width="377" alt="image" src="https://github.com/mate-academy/py-taxi-service-user-form-and-validation/assets/107141441/e91f16f8-3499-4cb2-a5d1-92475795ef6d">